<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deposit</class>
 <widget class="QWidget" name="deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deposit</string>
  </property>
  <widget class="QLineEdit" name="amount">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="term_years">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="interest_rate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tax_rate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Term (years)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Interest rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>66</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Tax rate</string>
   </property>
  </widget>
  <widget class="QComboBox" name="pereodic_pay">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>201</width>
     <height>37</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>every month</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>every quarter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>every day</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>161</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Periodicity of payments</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cap_int_checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Ð¡apitalization of interest    </string>
   </property>
  </widget>
  <widget class="QLabel" name="accures_int">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>320</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tax_amount">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>380</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="end_deposit">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>440</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>765</x>
     <y>380</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tax amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>320</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Accrued interest</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>440</y>
     <width>66</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Deposit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_result_deposit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>500</y>
     <width>103</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>result</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
